src/uf2.h

```diff
@@ -42,8 +42,12 @@ SOFTWARE.
 // Version is passed by makefile
 // #define UF2_VERSION         "0.0.0"
 
-#define CFG_UF2_FLASH_SIZE  (4*1024*1024)   // TODO absolute max flash size
-#define CFG_UF2_NUM_BLOCKS  0x10109         // just under 32MB
+#define CFG_UF2_FLASH_SIZE          (4*1024*1024)    // TODO absolute max flash size
+#define CFG_UF2_NUM_BLOCKS          (0x10109)        // just under 32MB
+#define CFG_UF2_SECTORS_PER_CLUSTER (1)
+// #define CFG_UF2_FLASH_SIZE          (256*1024*1024)  // TODO absolute max flash size
```

Comment:
Remove the commented-out defines.  Instead, if your intent is to define a highest value tested / supported, just make that a comment in the file.